### ISSUE TITLE
Loading a sacct data fails if non-time value exists in Timelimit column

### ISSUE THREAD

**Issue by jprorama** [NordicHPC/slurm2sql#3]
slurm2sql() will return an valueerror if the Timelimit column does not contain a value time value. 

```
$ slurm2sql testdb.sqlite3 -- -S 2020-09-08 -E 2020-09-15 -a 
Traceback (most recent call last):
  File "/home/jpr/.conda/envs/slurm-ds/bin/slurm2sql", line 8, in <module>
    sys.exit(main())
  File "/home/jpr/.conda/envs/slurm-ds/lib/python3.8/site-packages/slurm2sql.py", line 504, in main
    errors = slurm2sql(db, sacct_filter=sacct_filter,
  File "/home/jpr/.conda/envs/slurm-ds/lib/python3.8/site-packages/slurm2sql.py", line 650, in slurm2sql
    processed_line = {k.strip('_'): (COLUMNS[k](line[k])
  File "/home/jpr/.conda/envs/slurm-ds/lib/python3.8/site-packages/slurm2sql.py", line 650, in <dictcomp>
    processed_line = {k.strip('_'): (COLUMNS[k](line[k])
  File "/home/jpr/.conda/envs/slurm-ds/lib/python3.8/site-packages/slurm2sql.py", line 69, in slurmtime
    if len(hms) >= 1:   seconds += 60   * int(hms[-2] if len(hms)>=2 else hms[-1])  # min
ValueError: invalid literal for int() with base 10: 'Partition_Limit'
```

**Comment by jprorama** [comment]
The value of Timelimit can be set to "Partition_Limit" by slurm in cases where time_limit has no value.

https://github.com/SchedMD/slurm/blob/master/src/api/job_info.c#L485

With a string in the data the slurmtime() function errors out.

**Comment by rkdarst** [comment]
Fixed by #4.
